From: SeongJae Park <sj@kernel.org>
Date: Fri, 26 Dec 2025 11:18:17 -0800
Subject: [PATCH] mm/memory: mark faults_monitor_controls_lock as static

Mark it as static, as it doesn't need to be exposed.  Otherwise, sparse
complains.

Reported-by: kernel test robot <lkp@intel.com>
Closes: https://lore.kernel.org/oe-kbuild-all/202512261908.fhV5eWFj-lkp@intel.com/
Signed-off-by: SeongJae Park <sj@kernel.org>
---
 mm/memory.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mm/memory.c b/mm/memory.c
index 5987a67c53548..924f5fbb15085 100644
--- a/mm/memory.c
+++ b/mm/memory.c
@@ -7557,7 +7557,7 @@ void vma_pgtable_walk_end(struct vm_area_struct *vma)
 }
 
 static LIST_HEAD(faults_monitor_controls);
-DEFINE_SPINLOCK(faults_monitor_controls_lock);
+static DEFINE_SPINLOCK(faults_monitor_controls_lock);
 
 void faults_monitor_register(struct faults_monitor_control *control)
 {
-- 
2.47.3

